internal/runners/languages/languages.go

```diff
@@ -51,8 +68,41 @@ func (l *Languages) Run() error {
 		return locale.WrapError(err, "err_fetching_languages", "Cannot obtain languages")
 	}
 
+	// Fetch resolved artifacts list for showing full version numbers.
+	// Note: any errors here are not fatal, and only some of them should be reported to rollbar.
+	var artifacts []artifact.Artifact
+	if rt, err := runtime.NewFromProject(l.project, target.TriggerLanguage, l.analytics, l.svcModel, l.out, l.auth, l.cfg); err == nil {
+		artifacts, err = rt.ResolvedArtifacts()
+		if err != nil && !errs.Matches(err, store.ErrNoBuildPlanFile) {
+			multilog.Error("Unable to retrieve runtime resolved artifact list: %v", errs.JoinMessage(err))
+		}
+	} else {
+		logging.Error("Unable to initialize runtime for version resolution: %v", errs.JoinMessage(err))
+	}
+	ns := model.NewNamespaceLanguage()
+
 	for i := range langs {
-		langs[i].Name = strings.Title(langs[i].Name)
+		name := langs[i].Name
+		version := langs[i].Version
+		constraints := langs[i].VersionConstraints()
+
+		langs[i].Name = strings.Title(name)
```

Comment:
This would break structured output I think.

Reply:
I think I know what you mean, but this is how it was previously.

Reply:
I think we should just print what the API returns. 

I didn't realize this is what we were doing before though. Feel free to disregard, what I care about is that structured output still works and includes the language name.

Reply:
I've made the change to print only what the Platform returns. In this case, users will see "python" for both plain and structured output. This is a change because they used to see "Python" for both.